examples/verify_tsqrtP_calibration.py

```diff
@@ -49,35 +49,30 @@
         fl.basename = f"amplitude = {amplitude:.1f}"
         if not d.get_units("t") == "s":
             psd.logger.info(
-                psd.strm(
-                    "units weren't set for the t axis or else I can't read them from the hdf5 file!"
-                )
+                "units weren't set for the t axis or else I can't read them from the hdf5 file!"
             )
             d.set_units("t", "s")
         d *= V_atten_ratio  # V at output of amplifier
         d /= np.sqrt(50)  # V/sqrt(R) = sqrt(P)
 
         # {{{ functions that streamline plotting the desired pulse length datasets
-        def switch_to_plot(d, thisj):
-            thislen = d.get_prop("programmed_t_pulse")[thisj] / 1e-6
+        def switch_to_plot(d, j):
+            thislen = d.get_prop("programmed_t_pulse")[j] / 1e-6
```

Comment:
I rolled back -- `j` or `k` etc are by definition an index you're iterating over, so the `this` is a little weird